knees4bees/flashcards-starter
# Feature/card class

## Is this a fix or a feature?
- Feature.

## What is the change?
- Creates the `Card` class with its constructor.

## What does it fix?
- N/A

## Where should the reviewer start?
- Open `Card.js` and review its contents.

## How should this be tested?
- Run `nmp test` in the terminal (from the project directory) and verify that all the tests in `Card-test.js` are passing.
